packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -1,16 +1,22 @@
 abbr {
-  border-bottom: 1px dashed cv('action', 'base');
+  border-bottom: 1px dashed cv('action', 'dark');
+  text-decoration: none;
 }
 
 em {
   font-style: italic;
-  > em { text-decoration: underline; }
+
+  > em {
+    text-decoration: underline;
+  }
 }
 
 strong {
   font-weight: 600;
 
-  > strong { text-decoration: underline; }
+  > strong {
```

Comment:
Are nested `strong`s allowed/intended in the same way as `em`?

Reply:
They sure can!

See: [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong](MDN <strong>: The Strong Importance element) > Em vs Strong